### DISCUSSION TITLE
Registeration problem in vercel account

### DISCUSSION THREAD

**Discussion by Faizan-Iqbal-07** [vercel/community#4249] [Account Issue] [answered]
### Question

I have registration problem with vercel account. Please tell me how do i solve this problem

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by Faizan-Iqbal-07** [comment]
![Vercel](https://github.com/vercel/community/assets/114811615/313e3f3a-cfc4-4322-85b7-8eda61b3ff3e)
When I try to log in to vercel account , I get a message your account is blocked . I'm a student pleases help me to solve this problem as urgent as possible .its a request
My email is ; faizaniqbal2911@gmail.com

**Comment by Faizan-Iqbal-07** [comment]
5 days to go , Still i'm unable to open my vercel account ?

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. 

We all appreciate your patience as the team works through a large number of cases. For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Comment by Faizan-Iqbal-07** [comment]
Any update on this? 9 days to go .Still i'm unable to open my account

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received. I do not know when your case will get a response, but I do know the team is working through an unusually high number of cases and you can expect to wait a little longer than normal. For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Comment by Faizan-Iqbal-07** [comment]
It's been one month, and my account still hasn't been unblocked. Is there any specific reason. How much more time will it take?

**Reply by amyegan** [reply]
Sorry for the long wait. If you share the case number here, I can check on your ticket 🙇

**Comment by Faizan-Iqbal-07** [comment]
It has been more than a month, and I have made complaints, but you still haven't resolved my problem. Why are you wasting my time?

**Comment by Faizan-Iqbal-07** [comment]
#
It has been 2 months, and my account is still not unblocked. Will this issue be resolved, or are you not looking into it? @amyegan

**Reply by amyegan** [reply]
I don't know your case number. If you let me know the case number, I can look into it for you

**Reply by Faizan-Iqbal-07** [reply]
case number = 00165021.
 I submitted a complaint on September 26th. Please check the case number and let me know whether my account will be unblocked or not @amyegan
